src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Todo = props => {
+    return (
+        <div className={`confusingTodoDiv ${props.secondConfusingTodoFile.completed ? 'completed' : ''}`}
```

Comment:
While I appreciate your names here It might be better to use names related to what the data is.